components/dropdownlist/overview.md

```diff
@@ -44,29 +44,33 @@ Selected value: @selectedValue
 }
 ````
 
+>caption The rendered DropDownList component from the code snippet above:
+
+![Blazor DropDown List component](images/dropdownlist-basic-screenshot.jpg)
+
 ## Data Binding
 
-The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#data-binding) [Read more about the Blazor DropDownList data binding...]({% slug components/dropdownlist/databind %}).
+The Blazor dropdown list @[template](/_contentTemplates/dropdowns/features.md#data-binding) [Read more about the Blazor dropdown list component data binding...]({% slug components/dropdownlist/databind %}).
```

Comment:
This contradicts the model we are currently following - using the actual component name. If I properly understand, we should use the separate keywords "dropdown" and "list" but I suggest not plugging them in every component mention.

Reply:
Concerns addressed through ea8366c9a0b2aeeeada1d9fc662ed794a55740b9